pkg/cloud/openstack/cluster/actuator.go

```diff
@@ -87,20 +87,41 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster) error {
 		return err
 	}
 
+	secGroupService, err := clients.NewSecGroupService(client)
+	if err != nil {
+		return err
+	}
+
 	// Load provider config.
 	_, err = providerv1.ClusterSpecFromProviderSpec(cluster.Spec.ProviderSpec)
 	if err != nil {
 		return errors.Errorf("failed to load cluster provider config: %v", err)
 	}
 
 	// Load provider status.
-	_, err = providerv1.ClusterStatusFromProviderStatus(cluster.Status.ProviderStatus)
+	providerStatus, err := providerv1.ClusterStatusFromProviderStatus(cluster.Status.ProviderStatus)
 	if err != nil {
 		return errors.Errorf("failed to load cluster provider status: %v", err)
 	}
 
 	// Delete other things
 
+	if providerStatus.GlobalSecurityGroup != nil {
+		klog.Infof("Deleting global security group: %v", providerStatus.GlobalSecurityGroup.Name)
```

Comment:
If it is a name, please use `security group %q` instead of `security group: %v`. `%q` will add quotes automatically to the name and no need for `:`.

Reply:
@xunpan thanks for your comments, done.